src/os/bluestore/bluestore_tool.cc

```diff
@@ -572,6 +577,27 @@ int main(int argc, char **argv)
       exit(EXIT_FAILURE);
     }
   }
+  if (action == "trim") {
+    if (path.empty()) {
+      cerr << "must specify bluestore path" << std::endl;
+      exit(EXIT_FAILURE);
+    }
+    if (really.empty() || strcmp(really.c_str(), "--yes-i-really-really-mean-it") != 0) {
+      cerr << "Trim must to confirm, --yes-i-really-really-mean-it" << std::endl;
```

Comment:
Suggest to rephrase to something like "Trimming is a dangerous operation which could cause data loss, please confirm with --yes-i-really-really-mean-it option".

Reply:
done